### PR TITLE
Update user_storage_ids migration to handle error "table already exists"

### PR DESCRIPTION
**Pull request by maureensturgeon**
The issue with the migration of user_storage_ids from pegasus to dashboard is that for environments built from scratch (such as when a new developer onboards or creating an adhoc) the migrated table in dashboard will be created by the schema before the pegasus migrations are run, so when the migrations are run, the table already exists. In this case, when there is no data that needs to be migrated, we can just drop the old table.

We already knew that this was necessary for the CI env where the database is always built from scratch, but we didn't think of the other scenarios.

## Testing story
Tested locally that the error was caught when the table already existed and that the query for count works as expected

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/migrations/137_rename_user_storage_ids.rb [hunk 1]
```diff
@@ -5,14 +5,29 @@
...
-      # This view exists as a safe-guard during the time between when the table rename is applied
-      # And the code referencing the new name is deployed, during that time queries to the old table name
-      # will hit this view and query from the renamed table
-      create_view(pegasus_user_storage_ids, DB[dashboard_user_project_storage_ids].select_all)
+        # This view exists as a safe-guard during the time between when the table rename is applied
+        # And the code referencing the new name is deployed, during that time queries to the old table name
+        # will hit this view and query from the renamed table
+        create_view(pegasus_user_storage_ids, DB[dashboard_user_project_storage_ids].select_all)
+      rescue Sequel::DatabaseError => e
+        is_table_already_exists_error = e.message.include? "Table 'user_project_storage_ids' already exists"
```
Comment (maureensturgeon) [inline pegasus/migrations/137_rename_user_storage_ids.rb:18]: I'm assuming that this message will be the same in all environments

File: pegasus/migrations/137_rename_user_storage_ids.rb [hunk 2]
```diff
@@ -5,22 +5,29 @@
 
     unless [:production].include?(rack_env) || ENV['CI']
```
Comment (jamescodeorg) [inline pegasus/migrations/137_rename_user_storage_ids.rb]: Do we need to remove the `CI` check here too?

Comment (maureensturgeon) [reply pegasus/migrations/137_rename_user_storage_ids.rb]: Yes! Good catch

**Review by jamescodeorg — APPROVED** [review]
Looks good to me!

**Review by sureshc — APPROVED** [review]
Sorry I didn’t anticipate the Continuous Integration build scenario!

**Pull request merged**